sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSources/AzureBlobDataFeedSource.cs

```diff
@@ -68,27 +69,9 @@ internal AzureBlobDataFeedSource(AzureBlobParameter parameter, AuthenticationTyp
             Container = parameter.Container;
             BlobTemplate = parameter.BlobTemplate;
 
-            SetAuthentication(authentication);
```

Comment:
`SetAuthentication` was one of the methods responsible for the conversion between authentication types (internal => public).